src/SprykerEco/Zed/Econda/Business/Collector/AbstractDatabaseCollector.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Econda\Business\Collector;
+
+use Generated\Shared\Transfer\BatchResultTransfer;
+use Generated\Shared\Transfer\LocaleTransfer;
+use Spryker\Service\UtilDataReader\Model\BatchIterator\CountableIteratorInterface;
+use Spryker\Service\UtilDataReader\Model\BatchIterator\PdoBatchIterator;
+use Spryker\Shared\SqlCriteriaBuilder\CriteriaBuilder\CriteriaBuilderInterface;
+use Spryker\Zed\Kernel\Persistence\QueryContainer\QueryContainerInterface;
+use SprykerEco\Zed\Econda\Business\Exporter\Writer\File\FileWriterInterface;
+use SprykerEco\Zed\Econda\Persistence\Econda\AbstractPdoEcondaQuery;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Output\OutputInterface;
+
+abstract class AbstractDatabaseCollector extends AbstractCollector implements DatabaseCollectorInterface
```

Comment:
Why to use collectors, they are old concept

Reply:
If these are not collectors, we should use other naming.

Reply:
What is the new concept?